aeon/dj_pipeline/utils/__init__.py

```diff
@@ -0,0 +1,5 @@
+def _make_maintenance_filter(position_df, maintenance_filter, start, stop):
+    """Make a boolean filter for eliminating maintenance period"""
+
+    maintenance_filter += (position_df.index >= start) & (position_df.index <= stop)
+    return maintenance_filter
```

Comment:
This is pretty much a one-liner code, I'd suggest not having this refactored as a separate `_make_maintenance_filter` function, but just copy that one line directly where you have it used.
I'd argue that it is more readable than having this refactored function.

Reply:
Yes, that makes sense. Removed that one.